recipes/go2/COPYING

```diff
@@ -0,0 +1,340 @@
+		    GNU GENERAL PUBLIC LICENSE
```

Comment:
Is this file not in the source archive?

Reply:
No, see this issue for all changes I'm requesting for the upstream package: https://bitbucket.org/DavidVilla/go2/issues/3/suggestions-to-improve-downstream